### PR TITLE
ci: Skip Secure Boot tests for SCOS

### PR DESCRIPTION
**Pull request by travier**
ci: Skip Secure Boot tests for SCOS

---

denylist: Skip Secure Boot tests for SCOS

### CODE REVIEW

File: ci/prow-entrypoint.sh [hunk 1]
```diff
@@ -117,9 +117,20 @@ cosa_build() {
 # Build QEMU image and run all kola tests
 kola_test_qemu() {
     cosa buildextend-qemu
-    cosa kola --basic-qemu-scenarios
+
+    # Skip Secure Boot tests on SCOS for now: https://github.com/openshift/os/issues/1039
```
Comment (jlebon) [inline ci/prow-entrypoint.sh:121]: Did you mean https://github.com/openshift/os/issues/1237 here?

**Review by jlebon — APPROVED** [review]

**Comment by travier** [comment]
Cross linking https://github.com/coreos/coreos-assembler/issues/3418

File: kola-denylist.yaml [hunk 2]
```diff
@@ -25,3 +25,13 @@
...
+
+- pattern: iso-live-login.uefi-secure
+  tracker: https://github.com/openshift/os/issues/1237
+  osversion:
+  - c9s
+
+- pattern: iso-as-disk.uefi-secure
+  tracker: https://github.com/openshift/os/issues/1237
+  osversion:
+  - c9s
```
Comment (jlebon) [inline kola-denylist.yaml:37]: Note that could be a single `pattern: *.uefi-secure` entry. But obviously fine as is too!

**Review by jlebon — APPROVED** [review]

**Comment by cgwalters** [comment]
LGTM though I probably would have said we should pin on a working kernel.

**Comment by dustymabe** [comment]
are the CI failures expected?

**Comment by travier** [comment]
The RHCOS CI failures are expected as we are in a mid state with RHEL 9.2.

The SCOS builds should pass. I agree that pinning would have been better but unfortunately C9S kernel get garbage collected quite quickly so that would mean another failure soon.

So skipping the Secure Boot still let's us test the rest of the kernel for now.

**Pull request merged**